bundles/org.eclipse.cdt.lsp.editor.ui/META-INF/MANIFEST.MF

```diff
@@ -3,7 +3,7 @@ Bundle-ManifestVersion: 2
 Bundle-Name: Language Server based C/C++ Editor UI
 Bundle-SymbolicName: org.eclipse.cdt.lsp.editor.ui;singleton:=true
 Bundle-Version: 1.0.0.qualifier
-Bundle-Activator: org.eclipse.cdt.lsp.editor.ui.LspEditorUiPlugin
+Bundle-Activator: org.eclipse.cdt.lsp.internal.clangd.editor.LspEditorUiPlugin
```

Comment:
Is it correct that the old editor.ui bundle shall be activated by the moved class?

Reply:
You are right here, I totally missed the Activator